.github/ISSUE_TEMPLATE/02-documentation.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Handsontable"
```

Comment:
```suggestion
name: "Handsontable Documentation"
```

Reply:
Done in 12deb8142e5897481b78119737521c3b0ff9ed8e.